packages/ecs-agent/Cargo.toml

```diff
@@ -25,8 +25,8 @@ url = "https://github.com/aws/amazon-ecs-cni-plugins/archive/db5864722987c34ba30
 sha512 = "550681f5cd9bdf46dd1a3353b9d328217a4d1b8697633fb70c19a0e2ddd839fb08bc764b63b1c23bc77b0d1c125ee8fbb5de0ad55628b1449261e9700e486387"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/aws/amazon-vpc-cni-plugins/archive/199bfc65cced4951cbb6a38e6e828afa8c2b023c/amazon-vpc-cni-plugins.tar.gz"
-sha512 = "ee1c2230c43fa7b8b9a25319bd334abfc08210ea8956ccc4525fe936b48b22b85df778823138d58687e847762b436daa0e0ba69c210a00b173f9b933f656386d"
+url = "https://github.com/aws/amazon-vpc-cni-plugins/archive/947afa12ff561ad073045bb6ae3cbe456a891397/amazon-vpc-cni-plugins.tar.gz"
+sha512 = "ce07e658838e15dc7646c4976245ba4077efe4e65850e5a4d9e94570237ce5ec7da012eb2c9f85632835c4e28d6844aadb09a10704b2694cd4efb37d5045bd1b"
```

Comment:
Why this particular commit or update? It looks like the ecs-agent repo is still on the older `199bfc6` commit.

Reply:
I removed it and kept what the `ecs-agent` uses